Dockerfile

```diff
@@ -1,12 +1,6 @@
-FROM hmcts/cnp-java-base:openjdk-8u191-jre-alpine3.9-2.0.1
-
-ENV APPLICATION_TOTAL_MEMORY 1024M
-ENV APPLICATION_SIZE_ON_DISK_IN_MB 41
-ENV JAVA_OPTS ""
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.0
```

Comment:
not sure if this change is enough?
the app insights agent needs to be copied into the docker image, and the app insights agent version needs to be set

also since you've set java 11 should the codebase be updated to java 11 as well?